doc/query.md

```diff
@@ -119,6 +195,86 @@ query.put("age", gt12);
 
 See below for supported operators (Selections -> Conditions).
 
+#### Text search
+
+A text clause may be used (if a text index exists) as part of a query to perform full text search term matching, phrase matching, prefix wildcard matching, etc.  A text clause can stand on its own as a query or can be part of a compound query (see below).  Text search supports SQLite Enhanced Query Syntax, see [SQLite full text query][ftsQuery] for more details on syntax that is possible with text search.
+
+[ftsQuery]: http://www.sqlite.org/fts3.html#section_3
+
+##### Restrictions
+
+- Only one text clause per query is permitted.
+- All clauses in a query must be satisfied by an index if that query contains a text search clause.
+- Both tokenizers, `simple` and `porter`, that come with text search by default are case insensitive.
+
+To find documents that make mention of the term `doctor books` use:
```

Comment:
"that mention [either/all] of the terms in `doctor books` use:"

Not sure if this is either or all, looks like all from your comment below.